examples/rust/components/http-client/wit/world.wit

```diff
@@ -0,0 +1,7 @@
+package wasmcloud:hello;
+
+world hello {
+  import wasi:http/outgoing-handler@0.2.0;
```

Comment:
Why not `include wasi:http/proxy@0.2.0;` instead?

Reply:
Basically my reason is the same as https://github.com/wasmCloud/wasmCloud/pull/2122#discussion_r1600250528. I am not trying to show how to implement the proxy world, I'm trying to show how you would use a client